.github/workflows/build.yml

```diff
@@ -0,0 +1,371 @@
+name: build
+
+on:
+  push
```

Comment:
Wondering if this means that the CRT GH actions will be triggered every time we push a branch? Just want to double-check my understanding on the process to start the CRT when we're ready for release.

Reply:
Yes, the build workflow will be kicked off on every push, and then depending on what's been set [here](https://github.com/hashicorp/consul-terraform-sync/pull/369/files#diff-d51ad13228611273f3804158eadd6b38eb7fe8eb46d5e84b94e6416a15c63709R11) as a release branch, the promotion pipeline will kick off after a successful build workflow on that branch (in [crt-common-workflows])(https://github.com/hashicorp/crt-workflows-common/actions). We are working on an onboarding guide at the moment and will share that with the team once it's ready. We'll also be scheduling a knowledge share session soon with the consul api team and we'll make sure to answer these questions more fully!

Reply:
Gotcha! Thanks for clarifying and pointing out where the release branch is set. We're planning to release today so will be sure to update that field. Looking forward to the session and on boarding guide 💞